.github/workflows/build.yaml

```diff
@@ -19,11 +19,6 @@ jobs:
           sudo snap install charmcraft --classic --revision 1349
       - name: Build ROCK
         run: |
-          app_version=$(yq '.version' rockcraft.yaml)
-          version=$(yq '(.version|split("-"))[0]' rockcraft.yaml)
-          base=$(yq '(.base|split(":"))[1]' rockcraft.yaml)
-          tag=${version}-${base}_edge
-          sed -i "s/${app_version}/${tag}/g" rockcraft.yaml
```

Comment:
forgive my lack of knowledge here. But why was this replaced?

Reply:
This is coming from @dragomirp fix for Mysql and related to changes in rockcraft https://github.com/canonical/charmed-mysql-rock/pull/36